### DISCUSSION TITLE
Unable to depend on multiple crates from the same workspace by path

### DISCUSSION THREAD

**Discussion by Heeten** [evcxr/evcxr#334] [Q&A]
When I try to depend on multiple crates from the same workspace using local paths, the first one loads. After that, invocations of `:dep` that reference other crates int hat workspace all fail with an error that looks like:
`Failed to load the project at /tmp/.tmpmZY8Rs/Cargo.toml`

Right now the way I am trying to do this is by having a `:dep` line for every crate I want to pull in with the path to the package Cargo.toml for each one. I tried using the path to the workspace but that gave me a different error.

**Comment by Heeten** [comment]
A workaround I'm using is to make a crate in the workspace that exposes all the other crates I want to access using `pub use ...`

**Comment by davidlattimore** [comment]
I just tried this and it worked for me. Perhaps the problem is something else not related to the workspace?
